disruption_py/machine/cmod/physics.py

```diff
@@ -685,10 +685,10 @@ def get_kappa_area(params: PhysicsMethodParams):
             A dictionary containing the calculated "kappa_area".
         """
         aminor = params.mds_conn.get_data(
-            r"\efit_aeqdsk:aminor", tree_name="_efit_tree", astype="float64"
+            r"\efit_aeqdsk:aout/100", tree_name="_efit_tree", astype="float64"
```

Comment:
at least in the scope of this method, let's keep adding comments for units...
I _might_ prefer having them before the line, for brevity, eg:
```
# [m] -- or even, more explicitly -- r0 [m]
r0 = params.mds_conn.get_data(r"\efit_aeqdsk:rmagx/100", tree_name="_efit_tree")
```
vs
```
r0 = params.mds_conn.get_data(
            r"\efit_aeqdsk:rmagx/100", tree_name="_efit_tree"
)  # [m]
```
your choice though!

Reply:
re
```
a = mds.get_data(r"\efit_aeqdsk:aout/100")
# vs.
a = mds.get_data(r"\efit_aeqdsk:aout") / 100
```
We would only be caching the expression, so in the first case we would cache the entire expression `"\efit_aeqdsk:aout/100"`. If we always divide `aout` by 100 any time we use it, it would be better to keep it in the expression to save us on that division. If we are using `aout` and not always diving by 100 `mds.get_data(r"\efit_aeqdsk:aout") / 100`. Though I don't know how much time caching a single division operation will really help us.

Reply:
it should be reasonable that reading `r"\efit_aeqdsk:aout/100"` or `r"\efit_aeqdsk:aout"` will be _two separate_ cache lookups, but I was wondering whether you were doing anything more specific than using the node path as the cache key -- maybe you had already planned for trivial parsing of major operations? eg `*`, `/`, `+`, `-`.

Reply:
I added units to most of the individual MDS calls in `cmod/efit.py` and `cmod/physics.py`. 
@gtrevisan How should we add units to the lines in `efit_cols`?

Reply:
naa, this is fine for now -- in the future we might want to use something like pint, or maybe try and fetch the units from MDSplus itself.